src/identity.js

```diff
@@ -322,6 +329,15 @@ export class Identity extends EventEmitter {
         document.cookie = 'SP_ID=nothing; expires=Thu, 01 Jan 1970 00:00:00 GMT; path=/; domain=.';
     }
 
+    /**
+     * Check if we need to use the ITP workaround for Safari versions > 12
+     * @returns {boolean}
+     */
+    _itpModalRequired() {
+        return document.requestStorageAccess &&
+            navigator.userAgent.includes('Version/12.');
```

Comment:
Sounds like something that will work for Safari 12, but stop working when v13 is released next year... Maybe we need a zooper-fancy regexp?

Reply:
Done